<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Save to DataBase</string>
  </property>
  <widget class="QComboBox" name="dialectCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>5</y>
     <width>235</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PostgreSQL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MySQL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Oracle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Microsoft SQL Server</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SQLite</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="hostportLE">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>95</y>
     <width>235</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="userLE">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>35</y>
     <width>235</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dbLE">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>125</y>
     <width>235</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLE">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>65</y>
     <width>235</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dialect:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>35</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>65</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>95</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Host:Port:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>125</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Database:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>190</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>155</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Table:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tableLE">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>155</y>
     <width>235</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
